docs/troubleshooting/harvester.md

```diff
@@ -67,6 +67,121 @@ Users can generate a support bundle in the Harvester GUI with the following step
 - Enter a useful description for the support bundle and click `Create` to generate and download a support bundle.
     ![](/img/v1.2/troubleshooting/harvester-sb-support-modal.png)
 
+### Retain Support Bundle File and Download Manually
+
+By default the support bundle file is generated, downloaded and deleted automatically after users click the `Create` button.
+
+There are also scenarios that users want to keep the support bundle files for some time or re-download the interrupted downloading.
+
+- Users fail to download the support bundle file due to the slow network performance, network disruption or other reasons, they want to re-download the support-file. Even though the support bundle file remains in the cluster, the Harvester WebUI does not have a place to download the file.
+
+- It takes a long time to generate the support bundle file, users want to reuse a previous support bundle file for quick troubleshotting.
+
+
+The workaround is:
+
+1. Break the automatic downloading.
+
+- Click `Generate Support Bundle` from Harvester WebUI.
+
+- When the progress is about 20%, close the web page.
+
+- Check support bundle status from kubectl.
+
+```
+# kubectl get supportbundle -A -oyaml
+apiVersion: v1
+items:
+- apiVersion: harvesterhci.io/v1beta1
+  kind: SupportBundle
+  metadata:
+    creationTimestamp: "2024-02-02T11:18:09Z"
+    generation: 5
+    name: bundle-htl5f
+    namespace: harvester-system
+    resourceVersion: "1218311"
+    uid: a3776373-05fe-4584-8a9a-baac3fa91bbf
+  spec:
+    description: sp1
+    issueURL: ""
+  status:
+    conditions:
+    - lastUpdateTime: "2024-02-02T11:18:38Z"
+      status: "True"
+      type: Initialized
+    filename: supportbundle_db25ccb6-b52a-4f9d-97dd-db2df2b004d4_2024-02-02T11-18-10Z.zip
+    filesize: 8868712
+    progress: 100
+    state: ready
+```
+
+When a supportbundle object has status `progress: 100`, the support bundle file is ready to be downloaded.
+
+2. Download and retain the support bundle file.
+
+- Download from curl or wget
+
+You need to manually construct the URL with `?retain=true` param to retain the support bundle file after downloading.
+
+```
+curl -k https://192.168.122.199/v1/harvester/supportbundles/bundle-htl5f/download?retain=true -o sb2.zip
+```
+
+Check the supportbundle object, it is kept for reusing.
+
+```
+# kubectl get supportbundle -A
+NAMESPACE          NAME           ISSUE_URL   DESCRIPTION   AGE
+harvester-system   bundle-htl5f               sp1           3h43m
+```
+
+- Download from web browser
+
+Manually input the URL `https://192.168.122.199/v1/harvester/supportbundles/bundle-htl5f/download?retain=true` into the web browser.
+
+When `?retain=true` param is not added, downloading the support bundle file will lead to the deletion of related resources.
```

Comment:
```suggestion
1. Download the file.

  1. Create the download URL.

  Example:
  `https://192.168.122.199/v1/harvester/supportbundles/bundle-htl5f/download?retain=true`

  :::note

  You must include the following information:
  - Resource name of the file 
  - Parameter `?retain=true`: If you do not include this parameter, resources related to the support bundle are deleted after the file is downloaded.
  
  :::

  1. Download the file using either command-line tools (curl or wget) or a web browser.

  Example:
  curl -k https://192.168.122.199/v1/harvester/supportbundles/bundle-htl5f/download?retain=true -o sb2.zip

  1. Verify that resources related to the support bundle were not deleted.

  Example:
  # kubectl get supportbundle -A
  NAMESPACE          NAME           ISSUE_URL   DESCRIPTION   AGE
  harvester-system   bundle-htl5f               sp1           3h43m
```

Reply:
Please add backticks to the code block.

Reply:
>   1. Create a download URL that includes the following information:
>  
>   - IP address of ???
>   - Resource name of the file
>   - Parameter `?retain=true`: If you do not include this parameter, resources related to the support bundle are deleted after the file is downloaded.
>
>    Example:
>    `https://{ip-address}/v1/harvester/supportbundles/bundle-htl5f/download?retain=true`

@w13915984028 I modified this step to address @Martin-Weiss comment about using a placeholder for the IP address. Please complete the description so that users know which IP address to include.